### DISCUSSION TITLE
西武ATSの開発について

### DISCUSSION THREAD

**Discussion by bellpepper0606** [winup-zhou/MetroAts#5] [Ideas]
西武ATSを当方で作成しようとしているのですが、こちらのプロジェクトの一部として参加させていただくことは可能でしょうか？
C++上で制作が進んでいるため、C#に移植して、AtsExに対応させる方向で検討しています。
ご検討お願いいたします。

**Comment by winup-zhou** [comment]
プラグイン開発への参加は大歓迎です！
私はたまたま西武ATSのことをあまり知らなくて、ここしばらく実装方法を考えていました。
このリポジトリをforkすることで、あなたのコードを貢献することができます！
